docs/tutorials/e2e/prerequisites/prerequisites.md

```diff
@@ -137,7 +137,11 @@ export NO_PROXY="localhost,127.0.0.1,::1"
 
 #### http (80)
 
-The port http (80) should not be used, but in case it will, you can apply the above hints just by replacing https by http.
+The port http (80) should not be used, but it will, you can apply the above hints just by replacing https by http.
+
+#### Port 8080
+
+Port 8080 will be used by the EDC, you will need the port 8080 once you want to exchnage data with a remote node.  
```

Comment:
I can't find anything in the tutorial that tells us to enable port 8080. Do you only do this on the VMs that you have provided? @JohannesARENA2036

Reply:
Hi Alex, you are right in the currenty version we do not need Port 8080, we needed it for the extended version during the Tractus-X Days, when we  intriduced a thrid EDC communicating fron outside. We may delete it now and add it later, as you like. Sonner or later we will need this port.

Reply:
port 8080 was removed